### ISSUE TITLE
gnomecast is freezing and converting unnecessarily

### ISSUE THREAD

**Issue by RhFortuna** [keredson/gnomecast#124]
Hello!

Often, when I use gnomecast, the software freezes, but two ffmpeg processes are still active in converting the video.

Why two? With the same functions.

![img1](https://user-images.githubusercontent.com/66008341/83340224-dc753e80-a2ab-11ea-9b1f-12096705389c.png)

The output is a single file anyway.

![img2](https://user-images.githubusercontent.com/66008341/83340251-3118b980-a2ac-11ea-9ffd-17c1d89a46d6.png)

But the strangest thing is that when I want to use the already converted file to avoid a new crash the gnomecast converts again to the same patterns that were already in the input file.

gnomecast 1.9.7
pip 20.1.1 (3.6)

I'm sorry if I made a mistake, I'm new to GitHub.
Thank you.

**Comment by keredson** [comment]
I've seen this before and thought I'd fixed.  I can't seem to replicate in 1.9.7.  I added some debug code and released 1.9.8.  Can you try?  You should see two sections like:

```
---------------------
 starting ffmpeg at:
---------------------
  File "/usr/lib/python3.8/threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "gnomecast.py", line 1137, in update_transcoders
    self.transcoder = Transcoder(self.cast, fmd, self.video_stream, self.audio_stream, lambda did_transcode=None: GLib.idle_add(self.update_status, did_transcode), self.error_callback, transcoder)
  File "gnomecast.py", line 306, in __init__
    traceback.print_stack()
```

**Comment by RhFortuna** [comment]
I didn't see this bug with mkv files, but now the software has updated to version 1.9.9.
Would you still like me to test something?

**Comment by RhFortuna** [comment]
I remembered the file I used and tried to use it again. The error was the same. v1.9.9

```
 starting ffmpeg at:
---------------------
und - eng StreamMetadata(index:0:0, codec:h264, title:und) AudioMetadata(index:0:1, codec:aac, title:eng, channels:6)
Transcoder /home/rhfor/Vídeos/Filmes/FvF.mp4
transcode, transcode_video, transcode_audio True False True
ffmpeg -i /home/rhfor/Vídeos/Filmes/FvF.mp4 -map 0:0 -map 0:1 -c:a ac3 -b:a 256k -c:v copy /var/tmp/gnomecast_pid30535_transcode_ijptxh6i.mp4
---------------------
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 1135, in update_transcoders
    self.transcoder = Transcoder(self.cast, fmd, self.video_stream, self.audio_stream, lambda did_transcode=None: GLib.idle_add(self.update_status, did_transcode), self.error_callback, transcoder)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 306, in __init__
    traceback.print_stack()
 starting ffmpeg at:
---------------------
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 1135, in update_transcoders
    self.transcoder = Transcoder(self.cast, fmd, self.video_stream, self.audio_stream, lambda did_transcode=None: GLib.idle_add(self.update_status, did_transcode), self.error_callback, transcoder)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 306, in __init__
    traceback.print_stack()
```

Now I'm not sure if this is really a bug. This file is large (2GB), with two processes running the program's graphical interface freezes, but everything continues to run in the terminal.

**Comment by RhFortuna** [comment]
I tried it twice and appeared only one process. But I can confirm that the output had no sound.

```
 starting ffmpeg at:
---------------------
  File "/usr/lib/python3.6/threading.py", line 884, in _bootstrap
    self._bootstrap_inner()
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 1135, in update_transcoders
    self.transcoder = Transcoder(self.cast, fmd, self.video_stream, self.audio_stream, lambda did_transcode=None: GLib.idle_add(self.update_status, did_transcode), self.error_callback, transcoder)
  File "/usr/local/lib/python3.6/dist-packages/gnomecast.py", line 306, in __init__
    traceback.print_stack()
```

**Comment by keredson** [comment]
i think this should be fixed now.  released in 1.9.11
